docs/src/pages/components/text-fields/UseFormControl.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import FormControl, { useFormControl } from '@material-ui/core/FormControl';
+import Input from '@material-ui/core/Input';
+import Box from '@material-ui/core/Box';
+import FormHelperText from '@material-ui/core/FormHelperText';
+
+const MyFormHelperText: React.FC = () => {
```

Comment:
We don't use `React.FC` because of [its problematic declaration](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/). `function MyFormHelperText() {}` is our preferred usage.

Reply:
That's good to know.